### ISSUE TITLE
Hyperparameters for Mamba Block

### ISSUE THREAD

**Issue by Karn3003** [OpenGVLab/video-mamba-suite#20]
Hi,
Can you please share the Hyperparameters for Mamba you have used along with learning rate, scheduler, weight decay, betas etc?? It's overfitting a lot for my task. Basically I want to know following hyperparamters:

        d_state=16,
        d_conv=4,
        expand=2,
        dt_rank="auto",
        dt_min=0.001,
        dt_max=0.1,
        dt_init="random",
        dt_scale=1.0,
        dt_init_floor=1e-4,
        conv_bias=True,
        bias=False,
